tests/helpers/test_runs_helper.py

```diff
@@ -0,0 +1,91 @@
+from cherrytrack.helpers.runs import get_run_info, get_automation_system_for_id, get_automation_system_run_for_id
+from pytest import raises
+
+
+def find_automation_system(run, automation_systems):
```

Comment:
If this function is test specific rather move it under that test to better understand the context.

Reply:
It might not be applicable here but it can be dangerous to have these type of functions here because they could have a bug which is not tested but they are required for testing, vicious circle! 🔁